nameko/messaging.py

```diff
@@ -98,6 +106,7 @@ def spam(self, data):
         """
         self.exchange = exchange
         self.queue = queue
+        self.serializer = serializer
```

Comment:
Quite right. I didn't spot this in the review on #244. There's no realistic situation where ``setup()`` isn't called.